viam-cartographer_test.go

```diff
@@ -208,228 +105,9 @@ func setupDeps(attr *slamConfig.AttrConfig) registry.Dependencies {
 			}
 			deps[camera.Named(sensor)] = cam
 		case "good_camera":
```

Comment:
[optional] If we decide to change this to `invalid_sensor` as in https://github.com/viamrobotics/viam-orb-slam3/pull/19#discussion_r1139357862, should we do the same here?